modules/concept-docs/examples/n1ql_query.java

```diff
@@ -0,0 +1,121 @@
+import com.couchbase.client.core.error.IndexExistsException;
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.ClusterOptions;
+import com.couchbase.client.java.Collection;
+import com.couchbase.client.java.Scope;
+import com.couchbase.client.java.env.ClusterEnvironment;
+import com.couchbase.client.java.json.JsonArray;
+import com.couchbase.client.java.json.JsonObject;
+import com.couchbase.client.java.manager.query.CreatePrimaryQueryIndexOptions;
+import com.couchbase.client.java.manager.query.CreateQueryIndexOptions;
+import com.couchbase.client.java.manager.query.QueryIndexManager;
+import com.couchbase.client.java.query.QueryOptions;
+import com.couchbase.client.java.query.QueryResult;
+import com.couchbase.client.java.query.QueryScanConsistency;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.UUID;
+
+public class n1ql_query {
+
+  String connectionString="localhost";
+  String username="Administrator";
+  String password="password";
+  String bucketName="travel-sample";
+  String id = "user::" + UUID.randomUUID().toString();
```

Comment:
NIT: When an object is concatenated to a string using `+` the compiler automatically calls `toString()... so this could just be

    String id = "user::" + UUID.randomUUID();